src/app/store/modules/accidents.js

```diff
@@ -9,5 +9,21 @@ export default {
     data: null
   },
   mutations,
-  actions
+  actions: {
+    ...actions,
+    async getProperty(context) {
```

Comment:
I realize this would mean changing this everywhere since it's shared but -- is `getProperty` really the most meaningful name for this? Maybe `getResource`?

Reply:
I like getResource. That is what I'm calling the boilerplate get/put/post/delete requests in `src/app/store/services/index.js`